### DISCUSSION TITLE
Doc versioning?

### DISCUSSION THREAD

**Discussion by ed2050** [zauberzeug/nicegui#4137] [Ideas / Feature Requests]
I get that the website docs cater to current version.  That's fine.  [Python docs](https://docs.python.org/3/) do the same by default.

However python docs also let you browse older versions of docs, for those who aren't running latest version or need to see what changed.  Is there any chance nicegui.io/docs can do similar?  

Current docs are missing some content.  For instance, [ui.open](https://nicegui.io/documentation/section_pages_routing#ui_open) doesn't even exist anymore.  If you need to use `ui.open` because `ui.navigate` doesn't exist or isn't suitable, website docs are useless.

(I know there are other ways to view older docs.  Website is less hassle though)

**Comment by falkoschindler** [comment]
Yes, that would be great, but also quite a challenge regarding the infrastructure.

Slightly alternative idea: We could start today to add version information to the documentation whenever changing some API. Like "Added in version 2.10.0", "Changed in version 2.11.0", ..., similar to https://docs.python.org/3/library/itertools.html.

**Reply by ed2050** [reply]
Yes that would help.  Thanks.
